src/mango/src/mango_cursor.erl

```diff
@@ -48,18 +48,12 @@
 create(Db, Selector0, Opts) ->
     Selector = mango_selector:normalize(Selector0),
     UsableIndexes = mango_idx:get_usable_indexes(Db, Selector, Opts),
-    case length(UsableIndexes) of
-        0 ->
-            AllDocs = mango_idx:special(Db),
-            create_cursor(Db, AllDocs, Selector, Opts);
```

Comment:
I'm probably missing something here but are we getting rid of the fallback to all_docs?

Reply:
No. The fallback happens in the mango_idx_special. The all_docs is added as an index. Previously what would happen is in the `maybe_filter_by_sort` if there were no sort fields defined it would return `{ok, []}`. That would then be returned to the above case statement which would then use the `mango_idx:special`. Now that check happens in the sort filtering inside the mango_idx_special.

Reply:
ok, I see it now, thanks for clarifying!